pkgs/by-name/ge/gerrit-queue/package.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule
+, lib
+, fetchFromGitHub
+}:
+
+buildGoModule {
+  pname = "gerrit-queue";
+  version = "unstable-2023-10-20";
+
+  src = fetchFromGitHub {
+    owner = "flokli";
+    repo = "gerrit-queue";
+    rev = "0186dbde15c9b11dc17b422feb74c842f6fa605a";
+    hash = "sha256-zXB5vre/Vr7UOyeMnf2RCtMKm+v5RENH7kGPr/2o7mI=";
+  };
+  vendorHash = "sha256-+Ig4D46NphzpWKXO23Haea9EqVtpda8v9zLPJkbe3bQ=";
+
+  meta = with lib; {
```

Comment:
You're missing the meta.maintainers tag. How should maintain this package? (:

Reply:
fixed.